README.md

```diff
@@ -98,22 +98,17 @@ See our [Configuration Guide](CONFIG.md) for various configuration options, incl
 Note that the validator also has a [batch processing mode](gtfs-realtime-validator-lib/README.md#batch-processing) - see the [**gtfs-realtime-validator-lib** README](gtfs-realtime-validator-lib/README.md).
  
 ## Docker
- 
-Pull the latest image from [Docker Hub](https://hub.docker.com/r/cutrusf/gtfs-realtime-validator):
-
-`docker pull cutrusf/gtfs-realtime-validator`
- 
+### Setup
+1. Download and install [Docker](https://docs.docker.com/get-started/)
+2. Pull the [latest Docker image for this project](https://github.com/orgs/MobilityData/packages/container/package/gtfs-realtime-validator). For example, `docker pull ghcr.io/mobilitydata/gtfs-realtime-validator` for the latest snapshot version of the validator.
 Or, build the image yourself using the following:
-
 `docker build -t gtfs-realtime-validator .`
 
 Then, run with:
 
-`docker run gtfs-realtime-validator`
```

Comment:
I'm having problems getting the container port to be accessible via the old method on Windows 10 with Docker Desktop app - I'm not sure if something changed since we originally did this. 

I suggest we change to what I added in this commit - https://github.com/MobilityData/gtfs-realtime-validator/pull/86/commits/ed9f6f832abc3715c9bec3965c9c8488acd7b770 - which works for me (unless someone know why the previous doesn't work or how to fix it). Also note that the full name of the container needs to be used as it is in this change.